content/cloud-computing/concepts/iaas/iaas.md

```diff
@@ -0,0 +1,20 @@
+---
+Title: 'IaaS'
+Description: 'Infrastructure as a service (IaaS) is a cloud computing model that offers remote computer, storage, and networking resources that are available on demand.'
+Subjects:
+  - 'Computer Science'
+  - 'Networking'
+Tags:
+  - 'Networking'
+  - 'Cloud'
+  - 'Infrastructure'
+CatalogContent:
+  - 'foundations-of-cloud-computing'
+  - 'paths/back-end-engineer-career-path'
+---
+
+**Infrastructure as a service (IaaS)** is a cloud computing model that offers remote computer, storage, and networking resources that are available on demand. These services are typically offered on a pay-as-you-go basis. The user is given control of virtual machines that are guests on a cloud server running a hypervisor such as Oracle VirtualBox or VMware ESXi. They are allowed to run arbitrary software on these virtual machines, including operating systems.
+
+IaaS is also provided with an [API](https://www.codecademy.com/resources/docs/general/api) to control the low-level details of the underlying network infrastructure: things such as backups, partitioning, security, physical resources, and so on. While the user does not manage or control the underlying cloud infrastructure, they have granular control over operating systems, storage, deployed applications, as well as some networking components such as host firewalls.
+
+The user is responsible for maintaining the virtual machines, including keeping operating systems and applications patched and up-to-date, and pays the cloud service only for the number of resources allocated and consumed.
```

Comment:
Break up into a list that the user is responsible for

Reply:
```suggestion
The user's responsibilities include the following:

- Maintaining the virtual machines.
  - This includes keeping operating systems and applications patched and up-to-date.
- Paying the cloud service only for the number of resources allocated and consumed.
```